pkg/collect/mssql.go

```diff
@@ -0,0 +1,115 @@
+package collect
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	_ "github.com/microsoft/go-mssqldb"
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+<<<<<<< HEAD
```

Comment:
@chasehainey you left some merge conflicts behind

Reply:
Thanks! I'm not sure why my editor didn't alert me that those were still there.